train.gemspec

```diff
@@ -41,4 +41,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "docker-api", "~> 1.26"
   spec.add_dependency "google-api-client", ">= 0.23.9", "< 0.35.0"
   spec.add_dependency "googleauth", ">= 0.6.6", "< 0.11.0"
+  spec.add_dependency "dotenv", "~> 2.7.6"
+  spec.add_dependency "facets", "~> 3.1.0"
+  spec.add_dependency "ipaddr", "~> 1.2"
```

Comment:
A small ask: maybe a comment noting that these are required for azure, or group them into their own section using a comment and some whitespace?